contracts/modules/TransferManager.sol

```diff
@@ -130,7 +130,6 @@ contract TransferManager is OnlyOwnerModule, BaseTransfer, LimitManager {
         }
     }
 
-    // TODO: We should inherit the OnlyOwnerModule implementation
```

Comment:
You probably tried the same things but I would have expected that we could have made `TransferManager.addModule()` call `OnlyOwnerModule.addModule(_wallet, _module);` instead of duplicating its code. Or alternatively, make `OnlyOwnerModule` be the last contract inherited by `TransferManager` (instead of `LimitManager`) and use `super.addModule(_wallet, _module);` in `TransferManager.addModule()`. But none of these solutions seemed to work for me. Is that a solidity bug or am I missing something?

Also, if we're going to duplicate `OnlyOwnerModule`'s code, let's change "BM: module is not registered" to "TM: module is not registered".

Reply:
It's because `addModule` is `external`, see Chris's explanation in this discussion https://github.com/ethereum/solidity/issues/3881#issuecomment-579456777
We have the option of removing `addModule` from the `IModule` interface, make it public and then use `super`. Atm it's not used off that interface so I could go down that route. What do you think?

Reply:
I personally think it is important to keep the `addModule` method in the `IModule` interface as it will protect us from future mistake.

Guaranteeing that every module can add another module combined with the fact that a wallet needs at least one module ensures the wallet can never be in a blocked state (in the sense of not being upgradable).

Reply:
Yet another way is to switch `IModule` from an `interface` to `abstract contract` which allows public functions with no implementation at the same time as mandating all functions to be implemented.

Reply:
Yes that works for me.

Reply:
We could keep `IModule` as-is but make `addModule` in `BaseModule` and `OnlyOwnerModule` be `public` instead of `external`.

Reply:
That's also an option. I don't feel strongly for either of the two. I'll just go with your version as it causes less changes to current state. Let me know if that's okay